packages/lib-runtime/package.json

```diff
@@ -1,14 +1,20 @@
 {
-  "name": "@openshift-org/dynamic-plugin-sdk-runtime",
+  "name": "@openshift/dynamic-plugin-sdk-runtime",
   "version": "0.0.0-alpha1",
+  "description": "Allows loading, managing and interpreting plugins at runtime",
   "scripts": {
     "prepack": "yarn build",
     "prepublish": "yarn test",
     "build": "rm -rf dist && tsc --outDir dist",
-    "test": "echo 'Write some tests!'"
+    "test": "echo 'Write some tests!'",
+    "lint": "yarn lint:run --ext .ts,.tsx ."
```

Comment:
Here we assume the lib packages will be written purely in TypeScript.

Reply:
I see no reason we would do anything else. I think OpenShift Console only has `.js` in there because of the legacy code before TS was added.